esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -464,8 +464,52 @@ void CTpWrapper::getTpEspServers(IArrayOf<IConstTpEspServer>& list)
     }
 }
 
+static Owned<IEspTpMachine> createLocalHostTpMachine(const char *path)
```

Comment:
non-standard way of dealing with owner transfer.
should return a regular pointer to IEspTpMachine, caller then owns.